joblibspark/backend.py

```diff
@@ -91,6 +91,18 @@ def __init__(self, **backend_args):
             self._ipython = get_ipython()
         except ImportError:
             self._ipython = None
+        self._spark_supports_resource_profile = hasattr(
+            self._spark_context.parallelize([1]), "withResources"
+        ) and not self._spark.conf.get("spark.master", "").startswith("local")
+        if self._spark_supports_resource_profile:
+            self._resource_profile = resource_profile
+        else:
+            self._resource_profile = None
+            if resource_profile is not None:
```

Comment:
if spark master is 'local', printing another error message like "spark local mode doesn't support stage-level scheduling."